lib/events/index.js

```diff
@@ -2109,7 +2131,9 @@ export default function (self) {
       const row = self.selections[rowIndex];
       if (row) {
         if (row[0] === undefined) return;
-        startCell = [rowIndex, row[0] < 0 ? row[1] : row[0]];
+        let isNumbersRowSelected = row[0] < 0;
```

Comment:
I now see `isNumbersRowSelected` might not present what it can be. Sorry, it was my time looking at the code there. I will use `row[0] < 0` as an inline statement instead.